src/models/driver.py

```diff
@@ -0,0 +1,29 @@
+import rethinkdb as r
+from remodel.models import Model
+from remodel.helpers import create_tables, create_indexes
+
+# import classes within the same directory
+from course import Course
+from section import Section
+
+
+def main():
```

Comment:
This is often `init()`, and with tornado if you wrap it in `@gen.engine` it will know to call it when the web app starts.